docs/api.md

```diff
@@ -148,6 +148,7 @@ Following is the supported API format for network transformations:
                  type: (enum) one of the following:
                      add_regex_if: add output field if input field satisfies regex pattern from parameters field
                      add_if: add output field if input field satisfies criteria from parameters field
+                     add_label_if: add output field to list of labels if input field satisfies criteria from parameters field
```

Comment:
It's something we should have raised earlier as it also concerns "add_regex_if" and "add_if",  but this doesn't look like a network-specific transformation, shouldn't that rather go in generic_transform?

Reply:
@jotak Should we do this in a separate PR? We would move add_if, add_regex_if, add_label_if to generic transform.

Reply:
yeah no problem with me to do that later

Reply:
@KalmanMeth if you can open an issue so we will not forget? This will be a "Breaking changes" one

Reply:
@KalmanMeth I see that the way you set the value in the code is `labels[rule.Output] = rule.c` --- can you extend the text to explain that output =key and Assignee=value